src/components/document/canvas.tsx

```diff
@@ -96,10 +96,14 @@ export class CanvasComponent extends BaseComponent<IProps> {
     const {content, document, ...others} = this.props;
     const documentContent = document ? document.content : content;
     const documentId = document?.key;
+    const documentTypeThemeClass = document?.type === "planning" ? "planning-doc" : "";
```

Comment:
Too long and too specific. Just go with `typeClass` here and elsewhere.
```suggestion
    const documentTypeThemeClass = document?.type === "planning" ? "planning-doc" : "";
```